pom.xml

```diff
@@ -354,39 +323,40 @@
   </reporting>
 
   <profiles>
-
     <profile>
-      <id>publish</id>
-
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-source-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>attach-sources</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-javadoc-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>attach-javadocs</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
I would like to use a profile name based approach, rather than switch to the use of properties. I think `activebydefault` works as expected in the form I created here.